src/components/DayPicker.jsx

```diff
@@ -59,6 +59,7 @@ const propTypes = forbidExtraProps({
   ...withStylesPropTypes,
 
   // calendar presentation props
+  customRef: PropTypes.element,
```

Comment:
a ref can be a callback, as well as a string, so i don't think this is the right propType for it